## Nature
* The 3'UTR is the mRNA sequence downstream of the stop-codon of a main-ORF
* Evolutionary conserved
* Longer than the 5'UTR -> several hundred to several thousand of nucleotides
	* Increased with evolution -> important role
* Contain proportionally fewer introns than 5'UTR
	*  explained by the induction of non-sense mediated decay by introns downstream of stop codons and the presence of splicing signals in 3’UTRs leading to negative selection
* Functions
	* Translational control
	* mRNA degradation
	* mRNA localization
## microRNAs
* Small RNA molecules (20nts) 
* bind to sequence motifs in the 3’UTR
	* binding sites also found in coding regions of mRNAs
* regulate gene expression post-transcriptionally by RNA-mediated interference (RNAi), resulting in altered mRNA- and protein-levels
* Mechanism:
	* Transcribed from a single transcription unit or from introns of their target mRNA
	* Exported to the cytoplasm and processed by endonucleolytic cleavage
	* Allowing target binding and assembling of the miRNA induced silencing complex (miRISC)
	* The miRISC then recruits the CCR4-NOT complex, which contains exonucleases to remove the polyA-tail of the mRNA (deadenylation)
	* Leading to subsequent removal of the 5’cap (decapping) by DCP2 and ultimate degradation of the transcript by XRN1 in 5’-3’ directionality
* mRNAs targeted are, besides degraded, also translationally repressed
	* 6-29% account for translational repression and
	* it is widely accepted that miRNAs repress cap-dependent translation
* Despite the idea that mRNA degradation is a consequence of translational inhibition by miRNAs, mRNAs can be degraded independently of translation of which the CCR4-NOT complex has been proposed to be translationally repressive in the absence of deadenylation
	* a wide role for CCR4-NOT beyond mRNA-translation and stability, including mRNA localization and codon-usage
* Translational repression has also been linked to DEAD-box helicases like eIF4A1/2 and DDX6, where structured 5’UTRs remain unresolved due to interference by miRISC or the interaction of NOT1, a member of the CCR4-NOT complex, with eIF4A2
*  Conversely, studies in Drosophila melanogaster have shown a release instead of a recruitment of eIF4A and eIF4E
* In contrast, mRNAs with unstructured 5’UTRs are unsusceptible to miRNA induced translational repression (miRNA silencing)
* mRNAs containing IRES, 
	* do not require scanning and are resistant to miRNA silencing, (supporting the model of interference with the scanning-mechanism of the 43 PIC)
	* In contrast IRES requiring eIF4A which can be repressed by miRNAs
* DDX6 has also been proposed to play a role in silencing since abolishing the interaction between DDX6 and NOT1 reduced the silencing ability and repressed translation independent of mRNA degradation
* Potentially, DDX6 represses translation via binding to 4E-T, a transport protein of eIF4E, which in turn competes for the interaction with eIF4G disrupting the eIF4E-eIF4G binding
	* This model is challenged by only partial loss of silencing when 4E-T is depleted 
* Moreover, DDX6 has been shown to not associate with miRNA targeted transcripts in a transcriptome-wide study, as opposed to loss of DDX6 in embryonic stem cells leading to up-regulation of miRNA targeted transcripts
