## Nature
* The 3'UTR is the mRNA sequence downstream of the stop-codon of a main-ORF
* Evolutionary conserved
* Longer than the 5'UTR -> several hundred to several thousand of nucleotides
	* Increased with evolution -> important role
* Contain proportionally fewer introns than 5'UTR
	*  explained by the induction of non-sense mediated decay by introns downstream of stop codons and the presence of splicing signals in 3’UTRs leading to negative selection
* Functions
	* Translational control
	* mRNA degradation
	* mRNA localization
## microRNAs
* Small RNA molecules (20nts) 
* bind to sequence motifs in the 3’UTR
	* binding sites also found in coding regions of mRNAs
* regulate gene expression post-transcriptionally by RNA-mediated interference (RNAi), resulting in altered mRNA- and protein-levels
* Mechanism:
	* Transcribed from a single transcription unit or from introns of their target mRNA
	* Exported to the cytoplasm and processed by endonucleolytic cleavage
	* Allowing target binding and assembling of the miRNA induced silencing complex (miRISC)
	* The miRISC then recruits the CCR4-NOT complex, which contains exonucleases to remove the polyA-tail of the mRNA (deadenylation)
	* Leading to subsequent removal of the 5’cap (decapping) by DCP2 and ultimate degradation of the transcript by XRN1 in 5’-3’ directionality
* mRNAs targeted are, besides degraded, also translationally repressed
	* 6-29% account for translational repression and
	* it is widely accepted that miRNAs repress cap-dependent translation
* Despite the idea that mRNA degradation is a consequence of translational inhibition by miRNAs, mRNAs can be degraded independently of translation of which the CCR4-NOT complex has been proposed to be translationally repressive in the absence of deadenylation
	* a wide role for CCR4-NOT beyond mRNA-translation and stability, including mRNA localization and codon-usage
* Translational repression has also been linked to DEAD-box helicases like eIF4A1/2 and DDX6, where structured 5’UTRs remain unresolved due to interference by miRISC or the interaction of NOT1, a member of the CCR4-NOT complex, with eIF4A2
*  Conversely, studies in Drosophila melanogaster have shown a release instead of a recruitment of eIF4A and eIF4E
* In contrast, mRNAs with unstructured 5’UTRs are unsusceptible to miRNA induced translational repression (miRNA silencing)
* mRNAs containing IRES, 
	* do not require scanning and are resistant to miRNA silencing, (supporting the model of interference with the scanning-mechanism of the 43 PIC)
	* In contrast IRES requiring eIF4A which can be repressed by miRNAs
* DDX6 has also been proposed to play a role in silencing since abolishing the interaction between DDX6 and NOT1 reduced the silencing ability and repressed translation independent of mRNA degradation
* Potentially, DDX6 represses translation via binding to 4E-T, a transport protein of eIF4E, which in turn competes for the interaction with eIF4G disrupting the eIF4E-eIF4G binding
	* This model is challenged by only partial loss of silencing when 4E-T is depleted 
* Moreover, DDX6 has been shown to not associate with miRNA targeted transcripts in a transcriptome-wide study, as opposed to loss of DDX6 in embryonic stem cells leading to up-regulation of miRNA targeted transcripts

## RNA Binding Proteins (RBPs)
*  Cis-elements consisting in short (3-8nts) and repeated sequence motifs as hubs for RBPs
* Diversified in mRNAs by alternative cleavage and polyadenylation of 3’UTRs. Often tissue-specific, leading to several 3’UTR variants of the same mRNA
* ARE elements
	* AU-rich
	* found in transcripts encoding for short-lived factors such as oncogenes, growth factors and cytokines
	* role in mRNA stability or translational regulation
	* Some ARE - RBPs of translational regulation:
		* **HuR** stabilize ARE-containing mRNAs by inhibiting the recruitment of the exosome complex to the mRNA
		* **TTP** destabilizes mRNAs by recruiting the exosome also shown to be a translational repressor by recruiting 4EHP, a cap-binding protein competing with eIF4E
		* **HuD** induces cap-dependent mRNA translation, thought to bind the polyA-tail and eIF4A in the closed loop formation
		* Drosophila examples:
			* Bicoid binds to a 3’UTR motif of the caudal mRNA and to 4EHP to repress translation
			* Cup, another eIF4E-binding protein, represses translation of the oskar mRNA via the 3’UTR-binding protein Bruno
			* To compensate the expression of X-chromosomal linked genes RBP Sxl binding to uridine-rich sequences in both 3' and 5' UTRs of the msl2 mRNA and inhibiting translation independent of 5’cap and polyA-tail, while impairing scanning by the 43S PIC
		* Xenopus examples:
			* CPEB, which binds to the CPE motif in 3’UTRs regulating their translation, associates with eIF4E through Maskin leading to translational repression similar to the mechanism of Cup in Drosophila
		* Other translation initiation down regulation is by inhibited association of the 43S PIC mediated by the GAIT-complex. The 3’UTR of the ceruloplasmin mRNA is bound by a multimeric complex, induced by interferon-γ, consisting of the ribosomal protein L13a including GAPDH, NS-associated protein 1 and Glu-Pro-tRNA-synthetase. This complex blocks interaction of eIF3 with eIF4G and impairs binding of the 43S PIC leading to decreased translation initiation despite a formation of the closed loop between PABP and eIF4G

## mRNA localization and transport
*  Mediated by RBPs into complexes of mRNA and protein – messenger ribonucleoproteins -> mRNPs
* mRNPs complexes are transported to cellular compartments by associating with motor proteins. 
	* Drosophila oocyte Oskar mRNA. Required to be translationally repressed by Bruno  and localized by microtubules through interactions with tropomyosin, Staufen and components of the exon junction complex (EJC)
* Mammalian cells mRNAs are transported via Zipcode-binding protein 1, binding to zipcode sequences in the 3’UTR. See the transport of beta-actin mRNA facilitated by KIF11, a motor-protein associated with tubulin
* Aggregation of translationally silent mRNAs considered responses to cellular stress:
	* Stress granules (SG)
		* associated with translational inhibition upon cellular stresses by phosphorylation of eIF2α. characterized by translation initiation factors
		* these have polyA-tails and can resume being translated
	* Processing-bodies (P-bodies)
		* linked to mRNA-silencing and decay, since the exonuclease XRN1 and silencing factors such as GW182, CCR4-NOT and Ago co-localize with these granules and mRNAs lack polyA-tails