State of the art that is the base of the project.

## Translation
The translations is a gene expression step that covers the decoding of mRNA to protein. "A post-transcriptional mechanism of gene expression regulation". It is responsible for the proteome. The rate of protein production per hour ranges widely between 10 to 10,000 proteins per mRNA in comparison to the production of 0.1 to 100 mRNAs by transcription. Illustrating the importance of mRNA translation regulation in mediating rapid changes to the **proteome, adaptive nature**. It is the most energy demanding process in the cell, up to 28% of the energy stored in ATP is consumed.

### Key concepts
* **Translational efficiency**: relative rate at which an mRNA is translated to protein, describes the proportion of mRNA  associated with ribosomes.

### Steps of translation
* Synthesis proteins
	* Initiation (rate limiting). Two mechanisms:
		* Prevalent: highly dependent on the 5’cap-structure and requires a plethora of initiation factors (eIFs) 
			* The eukaryotic eIF4E binding the 5’cap is the rate-limiting step	
		* Secondary: uses an internal ribosome entry site (IRES)
		* Others for a subset of mRNAs mediated by: 
			* **eIF3d**: 5' cap biding protein requiring a distal stem-loop for eIF4F-independent initiation 
			* **DAP5**: stimulating IRES-driven translation and, in combination with eIF3d, modulating translation in a cap-dependent manner without requiring the eIF4F complex controlling T-cell phenotypes
		* Regulated by eIFs and features and structures in the 5’UTR
	* Elongation HERE DOING!
		* 80S formation in start-codon
		* tRNA$_i$$^M$$^E$$^T$ recruitment
		* eEF1A enters the ribosome A-site with new tRNA+aa
		* rRNA of large ribosomal unit
		* stabilization of peptidyl-tRNA by eIF5
		* eEF2 translocates tRNAs in P and A site to E and P (E -> P -> A)
	* Termination (with ribosome recycling)
		* Translation is terminated by  eukaryotic release factors (eRFs) 1 and 3
		* eRF1 is tRNA-like recognises stop codon 
		* Ribosome splits in 40S and 60S by ABCDE1 mediated by eEF1
		* 40S is recycled by eIF3 + eIF1 and eIF1A
		* 60S binds eIF6 and prevents premature joining of 40S 
* Degradation proteins
	* ??

### Translational Control
### Important regulators
* Codons (64, 61 gives aa)
	* Start-codon (AUG - Methionine)
		* Context
	* End-codons
* Amino acids (20 aa, 9 essential)
	* AA extra: **Selenocysteine**, necessary in redox-reactions, encoded by a stop-codon, UGA, incorporated by the SECIS Element (a particular RNA motif in 3'UTR)  
* Anti-codons / tRNAs
* tRNAs modifications
* ncRNAs
	* miRNAs
	* rRNAs (80% of RNA cell abundance)
	* tRNAs
* mRNAs (uni-directional read 5' to 3')
	* 1000 - 3000 nucleotides long
	* Regions:
		* 5'cap structure (m7GpppN)
			* Trancription initiation, nucleus
		* 5'UTR
			* Transcription Factors start-sites (TSS)
		* ORF
			* Start codon (AUG)
			* Stop codon ()
		* 3'UTR (sites for)
			* RNA binding proteins (RBPs)
			* lncRNAs
			* microRNAs
		* 3' poly-A tail
			* Trancription end (nucleus, or cytoplasm in germ-line)
	* Extremes protect degradation by exonucleases
* mRNA closed loop during translation
	* Important for efficiency and selectivity
	* The interaction of the poly A binding protein (PABP) with eIF4G leads to closed loop formation and enhances translation 
	* Indeed, mRNAs lacking a polyA-tail, e.g. of replication dependent histones, harbor 3’UTR stem-loops, bound by RNA-binding proteins to facilitate closed-loop formation
* Ribosome (12 days spanlife)
	* 60% rRNAs
		* High stability and cover 60% of total transcription + 50% of RNA-polymerase II activity
	* 40% protein (40S + 60S = 80S)
		* High stability and abundance, controlled by translation 
		* mRNAs with 5'UTR TOP motif (terminal oligo pyrimidine tract)
	* > 1 ribosome = polysome = > 1 protein


## Transcription
The transcription is a gene expression step that covers the transforming of the DNA to RNA.

### Steps of trancription
- pre-mRNA
- Alternative splicing -> Spliceosome (snRNAs + proteins)
	- Removal of introns
	- Join of exons
- Mature mRNA isoforms
- Protein isoforms
### Important Regulators
* RNA-polymerases
	* RNA-pol I -> most rRNA
	* RNA-pol II -> mRNAs and miRNAs
		* Regulated by its C-terminal domain (CTD) associated to many kinases, and its phosphorylation pattern steers activity, i.e. 5' capping affects CAGE TSS profiling methods
	* RNA-pol III -> tRNAs and 5S rRNA
* Chromatin access (nucleosomes = DNA + RNA + proteins)
* Transcription Factors
	* Mostly initiates transcription
	* Associate to transcription start-sites (TSS)
* Histones modifying enzymes
	* Open and close chromatin access
* 
## mRNA transport from the nucleus 

* mRNA is capped and spliced
* Mediated by TREX and TREX-2 multiprotein complexes
* Via nuclear pores
* mRNA quality control
* Co-regulated with mRNA transcription and splicing
* mRNAs related to cell cycle progression and survival, such as ODC1, Cyclin D1 and c-myc require the cap-binding protein eIF4E including a 3’UTR-motif for transport unlike other mRNAs which, links mRNA transport and regulation of mRNA translation 


